ct/komga.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: madelyn (DysfunctionalProgramming)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+function header_info {
+clear
+cat <<"EOF"
+ _  __                           
```

Comment:
Wrong ascii, use this https://www.patorjk.com/software/taag/#p=display&f=Slant&t=Komga